src/epub/text/chapter-61.xhtml

```diff
@@ -80,7 +80,7 @@
 			<p>“And do you understand them?”</p>
 			<p>“They are always the same.”</p>
 			<p>“And they mean⁠—”</p>
-			<p>“ ‘Nothing new’; ‘You have an hour’; or ‘Tomorrow.’ ”</p>
```

Comment:
I think these should stay outside as they are not ending dialogue, these are words used as words